content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md

```diff
@@ -0,0 +1,67 @@
++++
+type = "news"
+title = "Media Release: The Haiku Project and Freewear launch Official Haiku Merchandise"
+author = "jt15s"
+date = "2021-12-01 12:17:33+11:00"
+tags = ["haiku", "software", "freewear", "merchandise", "donations"]
++++
+
+<p align="left">FOR IMMEDIATE RELEASE</p>
+<p align="left">December 01, 2021</p>
+
+# THE HAIKU PROJECT AND FREEWEAR LAUNCH OFFICIAL HAIKU MERCHANDISE
+The Haiku Project launched its official merchandise today in partnership with FreeWear, an online free and open-source software (FOSS) merchandising store and print shop. The partnership will allow The Haiku Project to offer a new way for people to financially contribute to the development of Haiku whilst getting a physical item in return for their gratitude. Donations greatly contribute to initiatives such as the recent development contract, which is allowing waddlesplash, a respected Haiku contributor, to make further general improvements to Haiku, such as system improvements, new and refined drivers and more. The partnership will mean that a portion of merchandise sales is donated to the Haiku Project, allowing The Project to continue funding both the development contract, pay for ongoing costs such as server hosting, and the chance to pursue other initiatives.   
+
+FreeWear has greatly simplified the process of selling merchandise, having built their own online store, and handling the printing and shipping processes as well as the process of donating a portion of the sales straight to the FOSS project/organisation. Moreover, most of the world can receive merchandise from FreeWear. A partnership with FreeWear does not involve any upfront costs from the FOSS project, as a portion of the sales pays from expenses incurred such as printing.   
+
+Any FOSS organisations who are interested in working with FreeWear can contact them through email at admin(@)freewear.org   
+
+A list of the available merchandise, as well as the prices in Euros (EUR) and US Dollars (USD), follows, along with an image showing all the available products. Note that prices in US Dollars vary depending on exchange rates and are indicative only.   
```

Comment:
Do we really need this here? I would just put a link to the Freewear store so people can check it out for themselves.

Maybe it's different for the press release to send to other places, but on the website we don't need all the info here.